source/common/grpc/transcoder_input_stream_impl.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "common/buffer/buffer_impl.h"
+#include "src/transcoder_input_stream.h"
```

Comment:
This is a strange include. I was looking for the interface and documentation, where can I find this?